runtime/src/block_cost_limits.rs

```diff
@@ -32,21 +32,18 @@ pub const INSTRUCTION_DATA_BYTES_COST: u64 = 140 /*bytes per us*/ / COMPUTE_UNIT
 lazy_static! {
     /// Number of compute units for each built-in programs
     pub static ref BUILT_IN_INSTRUCTION_COSTS: HashMap<Pubkey, u64> = [
-        (feature::id(), COMPUTE_UNIT_TO_US_RATIO * 2),
-        (incinerator::id(), COMPUTE_UNIT_TO_US_RATIO * 2),
-        (native_loader::id(), COMPUTE_UNIT_TO_US_RATIO * 2),
-        (solana_sdk::stake::config::id(), COMPUTE_UNIT_TO_US_RATIO * 2),
```

Comment:
above four entries are not programs, ref https://github.com/solana-labs/solana/pull/30639#issuecomment-1460277898